client/blocks/daily-post-button/test/index.jsx

```diff
@@ -32,11 +38,11 @@ describe( 'DailyPostButton', () => {
 					markPostSeen={ markPostSeen }
 				/>
 			);
-			expect( dailyPostPrompt.type() ).toBeNull();
+			expect( container.firstChild ).not.toBeInTheDocument();
```

Comment:
This feels a bit odd, because it seems like an implicit way to test if the parent has no children. Would it make sense to go with this instead:

```suggestion
			expect( container ).toBeEmptyDOMElement();
```

Reply:
Fixed via Fixed via https://github.com/Automattic/wp-calypso/pull/64607/commits/c4aa9e67e21bcf7f4bfc59dcc745ee3e7cd6fed6 👍🏻